commit eee229da2705127308109e3655ff8bbd41d82569
Author: Georg Lehmann <dadschoorse@gmail.com>
Date:   Fri Apr 4 20:48:48 2025 +0200

    winevulkan: Update to VK spec version 1.4.312.
    
    Note that this also removes VK_NV_cuda_kernel_launch,
    as the extension is still provisional and the updated vk.xml
    now reflects that.

--- a/dlls/winevulkan/make_vulkan
+++ b/dlls/winevulkan/make_vulkan
@@ -114,6 +114,7 @@ UNSUPPORTED_EXTENSIONS = [
     "VK_KHR_shared_presentable_image", # Needs WSI work.
     "VK_KHR_win32_keyed_mutex",
     "VK_NV_external_memory_rdma", # Needs shared resources work.
+    "VK_NV_external_compute_queue", # Has a new dispatchable handle
 
     # Extensions for other platforms
     "VK_EXT_external_memory_dma_buf",
